recipes/nifti_clib/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+make nifti
```

Comment:
Looks like `gcc` is hard-coded in the Makefile based on the error message, so please try using `make CC=${CC} nifti` instead.